### DISCUSSION TITLE
Connection to github.com closed by remote host.

### DISCUSSION THREAD

**Discussion by justclint** [community/community#31651] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, Ive just all of a sudden started getting `Connection to github.com closed by remote host.` when trying to push to my repo. After running with `GIT_SSH_COMMAND="ssh -vvv"` I get the following:

```
> validate:yaml
Validating yaml syntax for all custom modules and exported config...
debug3: receive packet: type 80
debug1: client_input_global_request: rtype keepalive@openssh.com want_reply 1
debug3: send packet: type 82
Iterating over fileset files.yaml...
debug3: receive packet: type 96
debug2: channel 0: rcvd eof
debug2: channel 0: output open -> drain
debug2: channel 0: obuf empty
debug2: chan_shutdown_write: channel 0: (i0 o1 sock -1 wfd 7 efd 8 [write])
debug2: channel 0: output drain -> closed
debug3: send packet: type 1
debug1: channel 0: free: client-session, nchannels 1
debug3: channel 0: status: The following connections are open:
  #0 client-session (t4 r43 i0/0 o3/0 e[write]/0 fd 6/-1/8 sock -1 cc -1)

debug1: fd 0 clearing O_NONBLOCK
debug3: fd 1 is not O_NONBLOCK
Connection to github.com closed by remote host.  ############### Here is where disconnect happens.
Transferred: sent 3656, received 29028 bytes, in 20.7 seconds
Bytes per second: sent 176.9, received 1404.4
debug1: Exit status -1
```

Im dont know what to make of this data. Anyone know what this means or how to solve the issue?

Thanks!

**Comment by airtower-luna** [comment, marked as answer]
Actually this is where the close happens:

> `debug2: channel 0: rcvd eof`

GitHub sends you an EOF (End Of File), the stuff after if just cleaning up. I assume it has something to do with the YAML validation before that, but it's only partially visible, and definitely not standard SSH. Maybe a pre-receive hook or something? :thinking:

**Reply by justclint** [reply]
Thanks @airtower-luna , looks like something in our .git/pre-push script. Just commented that out and pushed fine.

**Comment by Dolo1993** [comment]
Hi, I am trying to get my file live on the internet in GitHub but finding it very difficult.  The below screenshot is what I continue to get. 
![fourth](https://user-images.githubusercontent.com/114083953/192281541-f646f7f5-468e-4158-b9b8-4e6a1bad7d93.png)
